### ISSUE TITLE
CDO Assignment Tasking

### ISSUE THREAD

**Issue by wongperson** [18F/State-TalentMAP#881]
As a CDO, 
I need a clear understanding of the things I need to do for a client on-assignment
so that I am able to more efficiently work with AOs to support assignees

### Acceptance criteria
- CDOs have a visualization of tasks to support assignees
- CDOs have a prioritization of tasks to help sort out what needs to be done first

**Comment by wongperson** [comment]
Closing as a duplicate of #885.
